pom.xml

```diff
@@ -47,7 +47,7 @@
     <sparkbundle.version>3.2</sparkbundle.version>
     <spark.version>3.4.2</spark.version>
     <sparkshim.artifactId>spark-sql-columnar-shims-spark32</sparkshim.artifactId>
-    <celeborn.version>0.3.0-incubating</celeborn.version>
+    <celeborn.version>0.3.2-incubating</celeborn.version>
```

Comment:
shall we add profile for 0.4 ?

Reply:
In theory, Celeborn only needs one version, as differences between versions have been eliminated through reflection.